lib/govuk_navigation_helpers/taxonomy_sidebar.rb

```diff
@@ -9,7 +9,7 @@ def initialize(content_item)
 
     def sidebar
       {
-        items: taxons
+        items: [taxons, elsewhere_on_govuk, elsewhere_on_the_web].flatten
```

Comment:
I personally prefer concatenation, because things like `flatten` make me wonder about the structure of these arrays you're joining. But fine to leave.